src/sentry/incidents/logic.py

```diff
@@ -1003,7 +1003,7 @@ def update_alert_rule(
     return alert_rule
 
 
-def update_detector(alert_rule: AlertRule, enabled: bool) -> None:
+def update_dual_written_detector(alert_rule: AlertRule, enabled: bool) -> None:
```

Comment:
should this method also be setting the `enabled` property on the detector since that's the field [currently being used](https://github.com/getsentry/sentry/blob/master/src/sentry/workflow_engine/processors/data_source.py#L26) to filter detectors? (at least until we figure out all the status stuff)

I wasn't sure if this was being invoked after that was set or what not.

Reply:
So https://github.com/getsentry/sentry/pull/93673/files was done incorrectly? Should I just change this all to `enabled` and not touch `status`?

Reply:
the field the platform uses is `enabled` -- from office hours yesterday it sounds like that might change ([thread](https://sentry.slack.com/archives/C07H2DHMSS0/p1750267686288159) discussing it as well).

I think for now, it's safest to set both -- until we can sort it all out.